src/details/ArborX_DetailsTags.hpp

```diff
@@ -45,6 +45,12 @@ struct Tag<Box>
   using type = BoxTag;
 };
 
+template <typename T>
+struct Tag<T>
```

Comment:
Hmm... It looks a little bit weird to treat `Point` and `Box` separately here. I would be happier if `Point` and `Box` would also have a `Tag` alias and we would not have to specialize here.

Reply:
Not going to happen here.